bundles/org.openhab.binding.mqtt.homeassistant/src/main/java/org/openhab/binding/mqtt/homeassistant/generic/internal/MqttBindingConstants.java

```diff
@@ -29,5 +29,5 @@
     // List of all Thing Type UIDs
     public static final ThingTypeUID HOMEASSISTANT_MQTT_THING = new ThingTypeUID(BINDING_ID, "homeassistant");
 
-    public static final String CONFIG_HA_CHANNEL = "mqtt:ha_channel";
+    public static final String CONFIG_HA_CHANNEL = "channel-type:homeassistant:mqtt-channel";
```

Comment:
The binding here is MQTT, so shouldn't it be `channel-type:mqtt:ha_channel`?

Reply:
Ok, I changed in favor of channel-type:mqtt:ha-channel and channel-type:mqtt:homie-channel